module/applications/actor/character-sheet.mjs

```diff
@@ -722,25 +721,26 @@ export default class CharacterActorSheet extends BaseActorSheet {
   async _getFavoriteData(type, id) {
     // Spell slots
     if ( type === "slots" ) {
+      const [method] = id.match(/^\D+/) ?? [];
```

Comment:
Seems like we should be able to use `actor.system.spells[id].type` to get the method rather than having to do regex on the ID.